Clarify that uploaders are invited, and must accept the invitation email (#2395)
dart-lang/site-www: src/tools/pub/cmd/pub-uploader.md

This command allows
[uploaders](/tools/pub/glossary#uploader) of a
package on the [pub.dev site]({{site.pub}}) to add (invite) or remove
other uploaders for that package. It has two sub-commands,
`add` and `remove`, that take the email address of the person to
add/remove as an uploader. For example:

```terminal
~/code/transmogrify$ pub uploader add bob@example.com
We have sent an invitation to bob@example.com, they will be added as uploader after they confirm it.

~/code/transmogrify$ pub uploader remove bob@example.com

> Successfully removed uploader from package.

Reviewer: I liked the previous text. Can we tweak this to include at least the email address? Something like this:

```suggestion
bob@example.com has been removed as an uploader for this package.
```

```terminal
$ pub uploader --package=transmogrify add bob@example.com

> We have sent an invitation to bob@example.com, they will be added as uploader after they confirm it.

Reviewer: Can we tweak this? It's >80 characters and not quite our style. Maybe:

```suggestion
We've sent an invitation email to bob@example.com.
They'll be added as uploader after they accept the invitation.
```

```

Note that uploaders are identified by their Google accounts, so use a Gmail or
Google Apps email address for any new uploaders.

> <aside class="alert alert-warning" markdown="1">
>   The `pub uploader add <email>` command will send an invite that the invited user has to accept.
>   For this to work, the `email` specified _must_ be the primary email of the associated Google account.
> </aside>

Reviewer: ```suggestion
{{site.alert.important}}
  The `pub uploader add <email>` command sends an invitation that
  the invited user must accept.
  For the invitation to work,
  `<email>` must be the **primary email address** of
  the associated Google account.
{{site.alert.end}}
```
